src/main/java/frc/robot/subsystems/GripperSystem.java

```diff
@@ -57,8 +61,10 @@ public CommandBase intake(){
         /** This logic changes the vision mode depending on whatever game piece has been grabbed */
         if(checkForCube()) {
           limelight.setPipeline(1);
+          aLedSubsystem.DriverColor(ColorType.PURPLE);
         } else if(checkForGamePiece()) {
           limelight.setPipeline(0);
+          aLedSubsystem.DriverColor(ColorType.YELLOW);
         }
```

Comment:
if possible this logic should be extracted to a command to avoid subsystems depending on other subsystems